reference/entry.json

```diff
@@ -148,6 +148,246 @@
+                          "enum": [
+                            "Available",
+                            "Unavailable"
+                          ]
+                        }
+                      },
+                      "required": [
+                        "type"
+                      ]
+                    }
+                  },
+                  "required": [
+                    "id"
+                  ]
+                }
+              },
+              "examples": {
+                "Updating the dates of the first subEvent": {
+                  "value": [
+                    {
+                      "id": 0,
+                      "startDate": "2021-05-17T22:00:00+00:00",
+                      "endDate": "2021-05-17T22:00:00+00:00"
+                    }
+                  ]
+                },
+                "Updating the status of the second subEvent": {
+                  "value": [
+                    {
+                      "id": 1,
+                      "status": {
+                        "type": "Unavailable",
+                        "reason": {
+                          "nl": "Afgelast wegens corona"
+                        }
+                      }
+                    }
+                  ]
+                },
+                "Updating the booking availability of the third subEvent": {
+                  "value": [
+                    {
+                      "id": 2,
+                      "bookingAvailability": {
+                        "type": "Unavailable"
+                      }
+                    }
+                  ]
+                },
+                "Updating multiple": {
+                  "value": [
+                    {
+                      "id": 0,
+                      "startDate": "2021-05-17T22:00:00+00:00",
+                      "endDate": "2021-05-17T22:00:00+00:00"
+                    },
+                    {
+                      "id": 1,
+                      "status": {
+                        "type": "Unavailable",
+                        "reason": {
+                          "nl": "Afgelast wegens corona"
+                        }
+                      }
+                    },
+                    {
+                      "id": 2,
+                      "bookingAvailability": {
+                        "type": "Unavailable"
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          },
+          "description": "The subEvents to update, with the properties to update. \n\nEach subEvent must have an `id` property to indicate which subEvent should be updated. This `id` is the position of the subEvent in the list of subEvents on the parent event. For example `0` for the first subEvent, `1` for the second subEvent, and so on.\n\nAll other properties are optional, and only properties that are included will be updated. No subEvents or properties will be removed."
```

Comment:
Maybe add something about the order based on the start date so it becomes clear how the indexes are calculated?

Reply:
Good point, added some more info 008793c